api/VERSION

```diff
@@ -1 +1 @@
-master
+2.3
```

Comment:
@nic-chen  Any reason to change this to `2.3`? IMHO, we shall keep using `master` in master branch.

Reply:
@imjoey  
it will show warning notification when dashboard version not matching APISIX(#1435).
so we should make it consistent with APISIX, use the last released version as the version number of the master. and we also have git commit hash to distinguish the specific build.

Reply:
> @imjoey
> it will show warning notification when dashboard version not matching APISIX(#1435).
> so we should make it consistent with APISIX, use the last released version as the version number of the master. and we also have git commit hash to distinguish the specific build.

@nic-chen OK, I see. the version here indicates the matched version of APISIX. Thanks.